examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/Src/main.c

```diff
@@ -527,10 +528,11 @@ static inline int8_t system_init( void )
     /* configure the timers  */
     Timer_Config();
 
-    UART_Configuration( WIFI_BOARD_UART_BAUDRATE );
+    UART_Configuration( WIFI_BOARD_UART_BAUDRATE ); /* From stm32_spwf_wifi.c */
 #ifdef USART_PRINT_MSG
     UART_Msg_Gpio_Init();
-    USART_PRINT_MSG_Configuration( DEBUG_UART_BAUDRATE );
+    if ( 0 > USART_PRINT_MSG_Configuration( DEBUG_UART_BAUDRATE ) )

```

Comment:
our general coding style is to always use { } even on single line returns.  BUT!  When it comes to platform-specific application wrappers, we follow the coding style of whatever we're running on / code that we're borrowing from.  Is this the standard for the example?